Library/Homebrew/system_config.rb

```diff
@@ -85,6 +85,19 @@ def describe_ruby
       end
     end
 
+    def describe_system_ruby
+      s = ""
+      case RUBY_VERSION
+      when /^1\.[89]/, /^2\.0/
```

Comment:
But currently for `describe_system_ruby`, the generic version and mac version only shares a partial logic. So I'm not sure how to make it work. Unless we want some thing like:
```ruby
def describe_system_ruby
  s = generic_describe_system_ruby
  if RUBY_PATH.to_s =~ %r{^/System/Library/Frameworks/Ruby.framework/Versions/[12]\.[089]/usr/bin/ruby}
    s.gsub! " => #{RUBY_PATH}", ""
  end
  s
end
```

Which I don't think it gets better.